xonsh/completers/tools.py

```diff
@@ -1,6 +1,9 @@
 """Xonsh completer tools."""
 import builtins
 import textwrap
+from typing import Callable, Optional, Set, Union
```

Comment:
the convention is to use `import typing as tp`

Reply:
This file is heavily typed, so I rather keep it like this for readability